src/components/view/ListView.vue

```diff
@@ -60,13 +60,22 @@
     </div>
     -->
 
+    <p v-if="actions.length > 0" :slot="actions.length > 0 ? 'expandedRowRender' : ''" slot-scope="record">
+      {{ record.id }}
+      <action-button
+        size="default"
+        :actions="$route.meta.actions"
+        :dataView="true"
+        :resource="record"
```

Comment:
@davidjumani for some reason only the first row of the table is used when records are clicked/passed -> perhaps need refactoring - let's discuss next week

Reply:
That was the case, so I changed the handleSubmit in Autogen.vue to read thre resource from the currentAction if present

Reply:
Changes made. Can test next week!